### DISCUSSION TITLE
how to create command with parameter

### DISCUSSION THREAD

**Discussion by ngdream** [telegraf/telegraf#1771] [Q&A] [answered]
hi
how to create create command with parameter like this :  /add nam)

**Comment by hypernova7** [comment, marked as answer]
```js
bot.command('add', ctx => {
  const [cmd, param] = (ctx.message.reply_to_message || ctx.message).text.split(' ')
  console.log(cmd, param)
  if (!param) return ctx.reply('Missing param');
  doSomething()
})
```

**Reply by ngdream** [reply]
> ```js
> bot.command('add', ctx => {
>   const [cmd, param] = (ctx.message.reply_to_message || ctx.message).text.split(' ')
>   console.log(cmd, param)
>   if (!param) return ctx.reply('Missing param');
>   doSomething()
> })
> ```
thanks
